src/main/java/org/broadinstitute/hellbender/tools/spark/bwa/BwaSpark.java

```diff
@@ -45,6 +48,12 @@ public boolean requiresReads() {
         return true;
     }
 
+    @Override
+    public List<ReadFilter> getDefaultReadFilters() {
+        // 1) unmapped or neither secondary nor supplementary and 2) has some sequence
```

Comment:
Comment seems a little off here -- shouldn't this say "primary" rather than "unmapped"?

Reply:
clarified